README.md

```diff
@@ -70,7 +70,7 @@ Put all data files to be uploaded in the same folder.
 ### Prepare manifest
 
 A manifest is a metadata (TSV) file that contains information about the data files to be uploaded. CLI tool will use this information to find, validate, and upload data files to Data Hub. There are 4 columns that are relevant to CLI tool:
-- the column containing file IDs
+- the column containing file IDs, unless you are updating existing data files and know the correct file IDs, manifest should not include this column, so that CLI will generate this column automatically. 
```

Comment:
so that can be removed.

Reply:
We can take a note to change it in 3.2.0. As I can't update 3.1.0 directly.